pkg/auth/signing/sign.go

```diff
@@ -127,3 +127,21 @@ func SignStreamID(ctx context.Context, signer Signer, unsigned *pb.SatStreamID)
 
 	return &signed, nil
 }
+
+// SignSegmentID signs the segment ID using the specified signer
+// Signer is a satellite
+func SignSegmentID(ctx context.Context, signer Signer, unsigned *pb.SatSegmentID) (_ *pb.SatSegmentID, err error) {
```

Comment:
what kind of `datatype/keyword` is that? such names are used in all methods in this file. its convinient to have it like that because in case of making another one it will have the same structure and naming. it's consistent and easy to modify copied (new) method.